src/status_im/ui/screens/screens.cljs

```diff
@@ -206,10 +205,6 @@
     :options   {:topBar {:visible false}}
     :component pin-messages/pinned-messages}
 
-   {:name      :photo-selector
```

Comment:
you need to add it to the new screens then?

Reply:
Photo selector is a bottom sheet. This was added there by mistake.